Neos.Flow/Classes/Reflection/DocCommentParser.php

```diff
@@ -47,10 +47,7 @@ public function parseDocComment($docComment)
 
         $lines = explode(chr(10), $docComment);
         foreach ($lines as $line) {
-            $line = trim($line);
-            if ($line === '*/') {
-                break;
-            }
+            $line = trim(str_replace('*/', '', $line));
```

Comment:
The only thing I fear happening is that some docblocks somehow might contain a `*/` somewhere with a meaning, hence I'd probably have used an approach to only remove it from the end of the line. But then again, how would that be a proper docblock? All I could find googling was https://stackoverflow.com/questions/59688653/i-need-to-use-a-splat-operator-within-a-php-docblock-and-cannot-figure-out-how-t which states that to "encode" a `*/` within a docblock, you should use `{@*}` so that would still be safe... dunno, I'm just overly precautious I guess 🙈

Reply:
That's a good point - let me `preg_replace` that instead.